### PR TITLE
added curated list of resources

### PR DESCRIPTION
**Pull request by javad94**
Feel free to change any part of it. Related to #5017

- [ ] Tests added / passed
- [ ] Passes `black dask` / `flake8 dask`

### CODE REVIEW

File: docs/source/educational-resources.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Educational Resources
+=======
```
Comment (TomAugspurger) [inline docs/source/educational-resources.rst]: I think this needs to match the length of the text.

**Comment by javad94** [comment]
@TomAugspurger Yes of course. I should put it in "Getting Started" toctree or "Help & reference"?

**Comment by TomAugspurger** [comment]
Not sure. Maybe "Help & Reference".

File: docs/source/educational-resources.rst (hunk 1 above)
Comment (javad94) [reply docs/source/educational-resources.rst]: Sorry, my bad. I'll fix it.

**Comment by javad94** [comment]
@TomAugspurger Ok then.

**Comment by javad94** [comment]
@TomAugspurger  Done.

**Comment by TomAugspurger** [comment]
Thanks!

**Comment by javad94** [comment]
@TomAugspurger Your welcome.

**Pull request merged**